src/condor_tests/CMakeLists.txt

```diff
@@ -299,6 +300,7 @@ if (BUILD_TESTING)
 			condor_pl_test(test_xfer_hold_codes "Test proper hold codes after xfer failure" "quick;ctest" CTEST DEPENDS "src/condor_tests/ornithology;src/condor_tests/conftest.py")
 			condor_pl_test(test_job_env "Test Job Environment" "quick;ctest" CTEST DEPENDS "src/condor_tests/ornithology;src/condor_tests/conftest.py")
 			condor_pl_test(test_bogus_collector "Test Bogus Collector" "quick;ctest" CTEST DEPENDS "src/condor_tests/ornithology;src/condor_tests/conftest.py")
+			condor_pl_test(test_startup_limits "Test startup throttling limits" "quick;ctest" CTEST DEPENDS "src/condor_tests/ornithology;src/condor_tests/conftest.py;${CMAKE_BINARY_DIR}/src/condor_tests/startup_limit_ctl")
```

Comment:
```suggestion
			condor_pl_test(test_startup_limits "Test startup throttling limits" "quick;ctest" CTEST DEPENDS "src/condor_tests/ornithology;src/condor_tests/conftest.py")
			add_dependencies_suffix_hack(test_startup_limits startup_limit_ctl.exe)
```

Reply:
My local cmake seems to be happy with the tweaks -- though my local cmake wasn't tripped up by this either.  Can you check BatLab?